sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -2476,4 +2476,14 @@ class SQLQuerySuite extends QueryTest with SharedSQLContext {
       assert(sql("SELECT * FROM array_tbl where arr = ARRAY(1L)").count == 1)
     }
   }
+
+  test("should be able to resolve a persistent view") {
```

Comment:
We don't define the behavior to resolve a view using a SQLContext in current master, this test case is to define that behavior.